Backendless/Push/Registrar.cs

```diff
@@ -19,9 +20,8 @@ internal static String RegisterDevice( String token, List<String> channels, Date
       deviceRegistration.Channels = channels;
       deviceRegistration.Expiration = expiration;
       deviceRegistration.DeviceToken = token;
-      deviceRegistration.DeviceId = Backendless.Messaging.DeviceID;
       deviceRegistration.Channels = channels;
-      deviceRegistration.Os = DeviceCheck.GetDeviceOS();
+      deviceRegistration.Os = DeviceCheck.GetDeviceOS();     

```

Comment:
all these things with mobile specific functionality must have conditional compiler statements, so this code doesn't break other builds

Reply:
All other projects doesn't have reference to this class